packages/base/any/kernels/5.10-lts/patches/series.arm64

```diff
@@ -16,3 +16,5 @@
 0020-accton-as5114.patch
 0021-tn48m-add-sfp-eeprom-support-to-ethtool.patch
 #0022-marvell-wa-tx-disable.patch
+0023-delta-tn48m-dn-series-dts.patch
+
```

Comment:
Please remove the blank line at the end of the file.

Reply:
Done and thanks.